src/components/Table/BlueprintTable.js

```diff
@@ -2,24 +2,71 @@ import React from "react";
 import PropTypes from "prop-types";
 import { Link } from "react-router-dom";
 import { useIntl } from "react-intl";
-import { TableComposable, Tr, Tbody, Td } from "@patternfly/react-table";
+import {
+  TableComposable,
+  Th,
+  Thead,
+  Tr,
+  Tbody,
+  Td,
+} from "@patternfly/react-table";
 import CreateImageWizard from "../Wizard/CreateImageWizard";
 import DeleteBlueprint from "../Modal/DeleteBlueprint";
 import ExportBlueprint from "../Modal/ExportBlueprint";
 
+const getLatestCreationDate = (images, blueprint) => {
+  const imagesByBlueprint = images.filter(
+    (image) => image.blueprint === blueprint.name
+  );
+  if (imagesByBlueprint.length > 0) {
+    const latestTimestamp = Math.max(
+      ...imagesByBlueprint.map((image) => image.job_finished)
+    );
+    return timestampToDate(latestTimestamp);
+  } else {
+    return "-";
+  }
+};
+
+const timestampToDate = (timestamp) => {
+  var options = { year: "numeric", month: "short", day: "numeric" };
+  var date = new Date(timestamp * 1000);
+  return date.toLocaleDateString("en-US", options);
+};
+
```

Comment:
In the `helpers.js` file we declare a `formTimestampLabel` function that is used in the images table. Can you call that here to keep the label consistent.

Reply:
Oops, didn't notice that one. Fixed!